homeassistant/components/openai_conversation/manifest.json

```diff
@@ -7,5 +7,5 @@
   "documentation": "https://www.home-assistant.io/integrations/openai_conversation",
   "integration_type": "service",
   "iot_class": "cloud_polling",
-  "requirements": ["openai==0.26.2"]
+  "requirements": ["openai==0.27.2"]
```

Comment:
Please add a link to a changelog or GitHub commit compare view for the version bump in the PR description. Thanks!

Reply:
Updated.